official/cv/segment-anything/point_inference.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+
+import cv2
+import numpy as np
+
+import mindspore as ms
+
+from segment_anything.build_sam import sam_model_registry
+from segment_anything.dataset.transform import TransformPipeline, ImageNorm, ImageResizeAndPad
+import matplotlib.pyplot as plt
+
+from segment_anything.utils.utils import Timer
+from segment_anything.utils.visualize import show_mask, show_box, show_points
+
+
+def infer(args):
+    ms.context.set_context(mode=args.mode, device_target=args.device)
+
+    # Step1: data preparation
+    with Timer('preprocess'):
+        transform_list = [
+            ImageResizeAndPad(target_size=1024, apply_mask=False, apply_point=True, apply_box=False),
+            ImageNorm(),
+        ]
+        transform_pipeline = TransformPipeline(transform_list)
+
+        image_path = args.image_path
+        image_np = cv2.imread(image_path) # (1024, 1024, 3)
+        image_np = cv2.cvtColor(image_np, cv2.COLOR_BGR2RGB)
+
+        # image = np.load(image_path)
+        # cv2.imwrite('./images/tumor1.png', image_np*255)
+        # image_npnew = cv2.imread('./images/tumor1.png')/255.0  # (1024, 1024, 3)
+        # diff = image_npnew - image_np
+
+        points_np = np.array([[[868, 622]], [[724, 466]], [[501, 543]], [[565, 505]], [[262, 456]]]) #  (bp, n, 2), in [w, h]
```

Comment:
is this point prompt bound with `images/tumor1.png`? 
what if user change `image_path` to another image?
better allow setting point prompt via argparser too.